### PR TITLE
Mobile modals part I: The base

### PR DESCRIPTION
**Pull request by madsnedergaard**
## Issue

https://linear.app/electricitymaps/issue/ELE-1340/[ui]-mobile-modals

## Description

- Adds basic components to prepares for adding full modals
- Also adds the two trigger buttons on mobile (_I had to change the buttons here as the same ones weren't available_)
- Removes Storybook example components

### Preview

_Left is old, right is new:_

Screenshot 2022-12-28 at 23 23 07

### CODE REVIEW

File: web/src/features/map-controls/MapControls.tsx [hunk 1]
```diff
@@ -83,7 +113,7 @@ export default function MapControls(): ReactElement {
   };
 
   return (
-    <div className="z-1000 pointer-events-none absolute right-3 top-3 hidden flex-col items-end md:flex">
```
Comment (tonypls) [inline web/src/features/map-controls/MapControls.tsx:86]: I may have been overzealous here 😆

**Review by tonypls — APPROVED** [review]
LGTM, I think the text color of the buttons for dark mode might need adjusting

**Comment by madsnedergaard** [comment]
> LGTM, I think the text color of the buttons for dark mode might need adjusting

Currently looks like this, what were you imagining? :)
Screenshot 2023-01-02 at 14 21 32

File: web/src/features/map-controls/MapControls.tsx (hunk 1 above)
Comment (madsnedergaard) [reply web/src/features/map-controls/MapControls.tsx:86]: Haha, apparently Tailwind only has up to `z-50` for some reason :D

**Pull request merged**